### DISCUSSION TITLE
Actualización a 1.11.26

### DISCUSSION THREAD

**Discussion by da-rodriguez** [chamilo/chamilo-lms#5284] [General] [answered]
Acabo de hacer la actualización desde 1.11.22, pero la página principal me sigue indicando que hay una actualización pendiente. Incluso tras haber limpiado el caché.

![version](https://github.com/chamilo/chamilo-lms/assets/5932634/3f668a56-6eef-4477-81a4-7517396a249e)

**Comment by NicoDucou** [comment, marked as answer]
Hola @da-rodriguez 

Este numero de version no se actualiza automaticamente.
Tienes que editar el archivo app/config/configuration.php y en la linea de 
`$_configuration['system_version'] = '1.11.22';`
Tienes que reemplazar por 
`$_configuration['system_version'] = '1.11.26';`

Saludos

**Comment by da-rodriguez** [comment]
Estuve mirando y no lo había visto.


Gracias

El vie, 22 mar 2024 a la(s) 8:48 a.m., Nicolas Ducoulombier (
***@***.***) escribió:

> Hola @da-rodriguez <https://github.com/da-rodriguez>
>
> Este numero de version no se actualiza automaticamente.
> Tienes que editar el archivo app/config/configuration.php y en la linea de
> $_configuration['system_version'] = '1.11.22';
> Tienes que reemplazar por
> $_configuration['system_version'] = '1.11.26';
>
> Saludos
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chamilo/chamilo-lms/discussions/5284#discussioncomment-8877173>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABNIMWUALVAXHXKJNR3IDD3YZQLADAVCNFSM6AAAAABFDE2366VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DQNZXGE3TG>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
________________________________________________

Daniel A. Rodriguez
Departamento de Tecnología para la Gestión
Escuela Provincial de Educación Técnica N° 1
Posadas - Misiones - Argentina
www.epet1.edu.ar
